templates/compose/cap.yaml

```
@@ -0,0 +1,79 @@
+# documentation: https://cap.so
+# slogan: Cap is the open source alternative to Loom. Lightweight, powerful, and cross-platform. Record and share in seconds.
+# tags: cap,loom,open,source,low,code
+# logo: svgs/cap.svg
+# port: 5679
+
+# Storage Configuration:
+# Option 1: Remote S3-compatible storage (AWS S3, Cloudflare R2, etc.)
+# Set these environment variables:
+# - CAP_AWS_ACCESS_KEY: Your S3/R2 access key
+# - CAP_AWS_SECRET_KEY: Your S3/R2 secret key  
+# - CAP_AWS_BUCKET: Your S3/R2 bucket name
+# - CAP_AWS_REGION: Your S3/R2 region (e.g., us-east-1, auto for R2)
+# - CAP_AWS_ENDPOINT: Your S3/R2 endpoint URL
+# - S3_PUBLIC_ENDPOINT: Public endpoint for your bucket (same as CAP_AWS_ENDPOINT for most cases)
+# - S3_INTERNAL_ENDPOINT: Internal endpoint (same as CAP_AWS_ENDPOINT for most cases)
+# - S3_PATH_STYLE: true for R2/most S3-compatible, false for AWS S3 virtual-hosted style
+#
+# Option 2: Local MinIO storage
+# Deploy MinIO as a separate service in the same network and set:
+# - CAP_AWS_ACCESS_KEY: MinIO root user
+# - CAP_AWS_SECRET_KEY: MinIO root password
+# - CAP_AWS_BUCKET: Your bucket name (e.g., capso)
+# - CAP_AWS_REGION: us-east-1 (or any region)
+# - CAP_AWS_ENDPOINT: http://minio:9000 (internal MinIO endpoint)
+# - S3_PUBLIC_ENDPOINT: http://your-minio-domain:9000 (public MinIO endpoint)
+# - S3_INTERNAL_ENDPOINT: http://minio:9000 (internal MinIO endpoint)
+# - S3_PATH_STYLE: true
+
+services:
+  cap-web:
+    container_name: cap-web
+    image: 'ghcr.io/capsoftware/cap-web:latest'
+    restart: unless-stopped
+    environment:
+      - SERVICE_URL_CAP_3000
+      - 'DATABASE_URL=mysql://$SERVICE_USER_MYSQL:$SERVICE_PASSWORD_MYSQL@cap-db:3306/${MYSQL_DATABASE:-planetscale}'
+      - 'WEB_URL=${SERVICE_URL_CAP_3000}'
+      - 'NEXTAUTH_URL=${SERVICE_URL_CAP_3000}'
```

Replace `SERVICE_URL_CAP_3000` with `SERVICE_URL_CAP` for WEB and NEXTAUTH URLs because we currently have a bug affecting some services env which has port number at the end - https://github.com/coollabsio/coolify/issues/6670#issuecomment-3322126426

The below suggestion fixes the app generating wrong login redirect url

```suggestion
      - 'WEB_URL=${SERVICE_URL_CAP}'
      - 'NEXTAUTH_URL=${SERVICE_URL_CAP}'
```